README.md

```diff
@@ -20,6 +20,8 @@ This is similar to the [Solution Packager](https://docs.microsoft.com/en-us/powe
 
 To get started, download and install the [Microsoft Power Platform CLI](https://docs.microsoft.com/en-us/powerapps/developer/data-platform/powerapps-cli).
 
+Easiest way to get pac cli is to use dotnet tool install: `dotnet tool install -g Microsoft.PowerPlatform.Cli`
```

Comment:
```suggestion
Easiest way to get pac cli is to use dotnet tool install: `dotnet tool install -g Microsoft.PowerApps.CLI.Tool`
```
Might also want to add a link to the nuget package? https://www.nuget.org/packages/Microsoft.PowerApps.CLI.Tool

Reply:
Fixed